Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM python:3.6.4-alpine3.7
+FROM python:2.7.14-alpine3.7
```

Comment:
Any reason for moving to Python 2?

Reply:
The new version doesn't work with python3.